### ISSUE TITLE
crash when importing gnotes folder

### ISSUE THREAD

**Issue by kerlerm** [giuspen/cherrytree#1686]
**Version, Operative system**
Linux x64, cherrytree PPA build 0.99.36

**Describe the bug**
Cherrytree crashes when trying to import gnotes from folder with 100 gnotes files in it.

Perhaps related to issue https://github.com/giuspen/cherrytree/issues/1593

**Dump**
```
==22830== Invalid read of size 8
==22830==    at 0x5FE972A: xmlpp::Node::get_first_child(Glib::ustring const&) (in /usr/lib/x86_64-linux-gnu/libxml++-2.6.so.2.0.7)
==22830==    by 0x442D01: CtTomboyImport::import_file(fs::path const&) (in /usr/bin/cherrytree)
==22830==    by 0x43C7F0: CtImports::traverse_dir(fs::path const&, CtImporterInterface*) (in /usr/bin/cherrytree)
==22830==    by 0x3D78B2: CtActions::_import_from_dir(CtImporterInterface*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (in /usr/bin/cherrytree)
==22830==    by 0x3D7ED2: CtActions::import_nodes_from_gnote_directory() (in /usr/bin/cherrytree)
==22830==    by 0x5DC6801: g_closure_invoke (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.6400.6)
==22830==    by 0x5DDA813: ??? (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.6400.6)
==22830==    by 0x5DE5BBD: g_signal_emit_valist (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.6400.6)
==22830==    by 0x5DE60F2: g_signal_emit (in /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0.6400.6)
==22830==    by 0x5508AB1: gtk_widget_activate (in /usr/lib/x86_64-linux-gnu/libgtk-3.so.0.2404.16)
==22830==    by 0x53D6A35: gtk_menu_shell_activate_item (in /usr/lib/x86_64-linux-gnu/libgtk-3.so.0.2404.16)
==22830==    by 0x53D6CC2: ??? (in /usr/lib/x86_64-linux-gnu/libgtk-3.so.0.2404.16)
==22830==  Address 0x8 is not stack'd, malloc'd or (recently) free'd
```

**Comment by giuspen** [comment]
could you try the latest from github for this issue?

**Comment by kerlerm** [comment]
Same for the latest self-compiled github binary. Problem persists.

**Comment by giuspen** [comment]
It would be useful if you could zip up the data that crashes for you and send it privately to me so I can reproduce. If possible, my email is in cherrytree help--about dialog

**Comment by giuspen** [comment]
will be fixed in 0.99.52
